db-mongo/hiCapSeeder.js

```diff
@@ -0,0 +1,41 @@
+const faker = require('faker');
+const { createReadStream, createWriteStream } = require('graceful-fs')
```

Comment:
Was there a specific reason to use graceful-fs instead of fs?  Not familiar with the package.

Reply:
Yes - I was receiving a specific error related to having too many open files at once and then the script would quit. The solution to that was using the graceful-fs package per Stack Overflow.